dev/io.openliberty.wsoc.internal_fat/test-applications/basic.war/src/basic/war/servlet/AddEndpointServlet.java

```diff
@@ -0,0 +1,76 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package basic.war.servlet;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.websocket.server.ServerContainer;
+import javax.websocket.server.ServerEndpoint;
+import javax.websocket.server.ServerEndpointConfig;
+
+import javax.websocket.OnMessage;
+import javax.websocket.OnOpen;
+import javax.websocket.Session;
+import javax.websocket.DeploymentException;
+
+import com.ibm.websphere.wsoc.WsWsocServerContainer;
+import basic.war.EchoServerEP;
+
+/**
+ * Servlet for testing removed restriction when adding an endpoint in Websocket 2.1 with addEndpoint methods in ServerContainer https://github.com/jakartaee/websocket/issues/211
+ * As stated in the specification before 2.1, "These methods are only operational during the application deployment phase of an application. Specifically, as soon as any of the
+ * server endpoints within the application have accepted an opening handshake request, the APIs may not longer be used. This restriction may 
+ * be relaxed in a future version."
+ * 
+ * Before 2.1, endpoints can only registered during the deployment of the web application. Once a websocket request enters the webcontainer,
+ * no more endpoints could be added with the API as stated in the spec "are only operational during the application deployment phase of an application. Specifically, as soon as any of the server endpoints within the application have accepted an opening handshake request, the apis may not longer be used. This restriction may be relaxed in a future version.". This restriction was removed in EE10 WebSocket 2.1
```

Comment:
Can you do this line over multiple lines? It seems super long relative to the rest of the javadoc

Reply:
Sure no prob I'll make the change